src/nova_tz/rsa_key.rs

```diff
@@ -0,0 +1,73 @@
+use crate::nova_tz::keyblob;
+use crate::nova_tz::Error;
+use crate::nova_tz::Result;
+use rsa::RsaPublicKey;
+use std::fs;
+use std::path::Path;
+use uuid::Uuid;
+
+pub struct TzRsaKeyInfo {
+    key_name: String,
+    rsa_key: RsaPublicKey,
+}
+
+impl TzRsaKeyInfo {
+    const NOVA_RSA_KEYS_PATH: &'static str = "/sys/nova-rsa/keys/";
+    const NOVA_RSA_ADD_PATH: &'static str = "/sys/nova-rsa/add";
+    const NOVA_RSA_REMOVE_PATH: &'static str = "/sys/nova-rsa/remove";
+
+    pub fn from_path(key_path: &str) -> Result<Self> {
```

Comment:
Why not take an actual `key_path: &Path` instead of a string?

Reply:
How this will help? `exists` is the only functionality needed from Path.

Reply:
Right and Path is he type of thing you're looking for. Better/clearer types..